packages/common/src/lib/common/exitCodes.ts

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const EXIT_CODES = {
+const EXIT_CODES: Record<string | number, string | number> = {
```

Comment:
This should be an `enum` instead, so we don't lose the syntax help for availabl properties.
`enum`s also include the reverse lookup, so that is no longer necessary.

Reply:
(This is the one TypeScript feature that has no equivalent in JS, but that I'd always miss if it were gone)

Reply:
does it work if you take the reverse lookup into account?

Reply:
doesnt seem so, because as soon as i define it as an enum it will be read-only